webpack.config.js

```diff
@@ -103,4 +102,32 @@ const storiesEditor = {
   },
 };
 
-module.exports = [storiesEditor];
+const dashboard = {
+  ...defaultConfig,
+  ...sharedConfig,
+  entry: {
+    'stories-dashboard': './assets/src/dashboard/index.js',
+  },
+  plugins: [
+    ...sharedConfig.plugins,
+    new WebpackBar({
+      name: 'Dashboard',
+      color: '#ade2cd',
```

Comment:
Why is the color hard coded?

Reply:
It's just some random color value for the terminal output. Don't think a variable makes sense here, as it's not used elsewhere.